docs/providers/aws/guide/serverless.yml.md

```diff
@@ -114,7 +114,7 @@ provider:
   httpApi:
     id: # If we want to attach to externally created HTTP API its id should be provided here
     name: # Use custom name for the API Gateway API, default is ${self:provider.stage}-${self:service}
-    payload: # Specify payload format version for Lambda integration (1.0 or 2.0), default is 1.0
+    payload: # Specify payload format version for Lambda integration ('1.0' or '2.0'), default is '1.0'
```

Comment:
I think these properties should have example value like other props.
Otherwise, the example serverless.yml is invalid YAML.

```suggestion
    payload: `1.0` # Specify payload format version for Lambda integration ('1.0' or '2.0'), default is '1.0'
```

Reply:
👍  that sounds good.

However in your code suggestion you use single backticks, not actual single quotes. That's a mistake?

Reply:
Ah, you are right.
It should be single quote.